install/kubernetes/cilium/templates/cilium-agent/clusterrole.yaml

```diff
@@ -72,29 +70,51 @@ rules:
 - apiGroups:
   - cilium.io
   resources:
-  - ciliumnetworkpolicies
-  - ciliumnetworkpolicies/status
+  - ciliumbgploadbalancerippools
+  - ciliumbgppeeringpolicies
+  - ciliumclusterwideenvoyconfigs
   - ciliumclusterwidenetworkpolicies
-  - ciliumclusterwidenetworkpolicies/status
+  - ciliumegressnatpolicies
   - ciliumendpoints
-  - ciliumendpoints/status
-  - ciliumnodes
-  - ciliumnodes/status
+  - ciliumendpointslices
+  - ciliumenvoyconfigs
   - ciliumidentities
   - ciliumlocalredirectpolicies
-  - ciliumlocalredirectpolicies/status
-  - ciliumegressnatpolicies
-  - ciliumendpointslices
-  - ciliumbgppeeringpolicies
-  - ciliumbgploadbalancerippools
+  - ciliumnetworkpolicies
+  - ciliumnodes
   verbs:
-  - '*'
+  - list
+  - watch
 - apiGroups:
   - cilium.io
   resources:
-  - ciliumclusterwideenvoyconfigs
-  - ciliumenvoyconfigs
+  - ciliumidentities
```

Comment:
@aanm I think we need `ciliumbgpeeringpolicies` here right? The `BGP Control Plane` performs and list and also a watch on those resources.